### PR TITLE
Update index.html

### PR DESCRIPTION
**Pull request by carsonicator**
I've added the new text; it just needs the links and indent added where indicated in our latest email exchange.

### CODE REVIEW

**Comment by galterlibrary** [comment]
Can one of the admins verify this patch?

**Comment by fenekku** [comment]
> it just needs the links and indent added where indicated in our latest email exchange.

@carsonicator Can you indicate those as comment(s) on GitHub, so I add the the right links / styling? There were a couple of emails exchanged and I want to make sure I get the right changes. Thanks!

**Comment by carsonicator** [comment]
Here are the links and indent info:

**Second block**:
'European Organization for Nuclear Research': https://home.cern/
'Zenodo': https://zenodo.org/

**Third block**:
'Next Generation Repositories': http://ngr.coar-repositories.org/

**Fourth block (indent)**:

> **Contact us.** Email us if you have questions or if you’d like to request a test user name and password for this site so you can create records of your own.

Links in block above:
'Email': invenioRDM@northwestern.edu

**Fifth block (indent)**:

> **Join us.** Get updates and test new releases on the product site, email us to request a demonstration or chat, take a look at our competitive landscape analysis,  and get involved with collaboration and discussion through our open source project. 

Links in block above:
'product site': https://invenio-software.org/
'open source project': https://github.com/inveniosoftware/invenio-app-rdm

File: cd2h_repo_project/modules/frontpage/templates/cd2hrepo_frontpage/index.html [hunk 1]
```diff
@@ -30,16 +30,27 @@
...
+    </p>
+    
     <br>
     <p class="footnote">
-      This project is funded in part by the <a href="https://ctsa.ncats.nih.gov/cd2h/">CTSA National Center for Data to Health (CD2H)</a> and supported by the NIH National Center for Advancing Translational Sciences, Grant Number U24TR002306, to help guide the cultural and technological changes necessary for data and informatics to improve research and health care.
+      InvenioRDM is powered by the <a href="https://galter.northwestern.edu/">Galter Health Sciences Library & Learning Center</a> and supported in part by the Feinberg School of Medicine and the NIH National Center for Advancing Translational Sciences (NCATS) through grants U24TR002306 (<a href="https://ctsa.ncats.nih.gov/cd2h/">CD2H</a>) and UL1TR001422 (<a href="https://www.nucats.northwestern.edu/">NUCATS</a>).
```
Comment (fenekku) [inline cd2h_repo_project/modules/frontpage/templates/cd2hrepo_frontpage/index.html]: The footer has "Powered by Invenio" already. Which one do we want to keep, how do we want to reword it? Potential synonyms for this sentence are "built by" / "developed by" / "is the effort of"...

Comment (carsonicator) [reply cd2h_repo_project/modules/frontpage/templates/cd2hrepo_frontpage/index.html]: What about ["Built with", "Built on"] Invenio?

Comment (fenekku) [reply cd2h_repo_project/modules/frontpage/templates/cd2hrepo_frontpage/index.html]: Went with "Built on". This page can be previewed on stage now.

**Comment by carsonicator** [comment]
@fenekku I made a couple of changes. Will you review again?

**Comment by fenekku** [comment]
@carsonicator you meant to merge this, right?

**Comment by carsonicator** [comment]
yes, sorry.

**Pull request merged**